package.json

```diff
@@ -383,6 +383,11 @@
             "require": "./dist/cjs/components/BarList/index.js",
             "module": "./dist/esm/components/BarList/index.js",
             "default": "./dist/esm/components/BarList/index.js"
+        },
+        "./Timeline": {
```

Comment:
well, we don't need add this on the package.json, because this combine the two json on the CI pipelines